userspace/libsinsp/sinsp.cpp

```diff
@@ -1141,6 +1142,9 @@ int32_t sinsp::next(OUT sinsp_evt **puevt)
 		m_firstevent_ts = ts;
 	}
 
+	// process asynchronous tasks running in the background
+	tick();
```

Comment:
I think it would be better to wrap this call so it only runs every hundreds of ms or so. I know you wrap the lower call in container manager, but I think it's best to wrap the top level one so we don't call all the tick methods for every syscall. Maybe it's finally time to bring run_on_interval from the other repo?

Reply:
yeah, run_on_interval seems the way to go